build/debian-iptables/Makefile

```diff
@@ -16,7 +16,7 @@
 
 REGISTRY?="staging-k8s.gcr.io"
 IMAGE=debian-iptables
-TAG=v10
+TAG=v11
```

Comment:
Has this image been pushed?
```
$ docker pull k8s.gcr.io/debian-iptables-amd64:v11
Error response from daemon: manifest for k8s.gcr.io/debian-iptables-amd64:v11 not found
```

Reply:
this one is trickier, since v10.1 makes no sense. v10-r2? bleh.

Reply:
though we did have a v5.1. (also a v5, and a v5.0?)